src/App.tsx

```diff
@@ -1,14 +1,80 @@
-/* eslint-disable max-len */
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
+import { getTodos, getUser } from './api';
+
+import { Todo } from './types/Todo';
+import { User } from './types/User';
+import { Options } from './types/Options';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
+import { filteredTodos } from './components/utils';
+
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [user, setUser] = useState<User | null>(null);
+  const [currentTodo, setcurrentTodo] = useState<Todo | null>(null);
+  const [isDataReady, setIsDataReady] = useState<boolean>(false);
+  const [isClickedOnTodos, setIsClickedOnTodos] = useState<boolean>(false);
+  const [query, setQuery] = useState<string>('');
+  const [option, setOption] = useState<Options>(Options.ALL);
+
+  const loadTodos = async () => {
+    try {
+      const todosFromServer = await getTodos();
+
+      setTodos(todosFromServer);
+      setIsDataReady(true);
+    } catch {
+      // eslint-disable-next-line no-alert
+      alert('Sorry, there is no todos yet');
+      setIsDataReady(true);
+    }
```

Comment:
Look, you do setIsDataReady(true); in both cases. So, move it to the final stage
```
} finally {
  setIsDataReady(true);
}